antarest/study/business/aggregator_management.py

```diff
@@ -144,20 +223,98 @@ def _gather_all_files_to_consider(self) -> t.Sequence[Path]:
         ]
         return all_output_files
 
+    def _gather_all_files_to_consider__all(self) -> t.Sequence[Path]:
+        # Links / Areas ids filtering
+        areas_or_links_ids = self._filter_ids(self.mc_all_path / self.output_type)
+
+        # Frequency and query file filtering
+        folders_to_check = [self.mc_all_path / self.output_type / id for id in areas_or_links_ids]
+        filtered_files: t.Dict[str, t.MutableSequence[str]] = {}
+        for folder_path in folders_to_check:
+            for file in folder_path.iterdir():
+                if file.stem == f"{self.query_file.value}-{self.frequency}":
+                    filtered_files.setdefault(folder_path.name, []).append(file.name)
+
+        # Loop to return the whole list of files
+        all_output_files = [
+            self.mc_all_path / self.output_type / area_or_link / file
+            for area_or_link, files in filtered_files.items()
+            for file in files
+        ]
+        return all_output_files
+
+    def columns_filtering(self, df: pd.DataFrame, is_details: bool) -> pd.DataFrame:
+        # columns filtering
+        lower_case_columns = [c.lower() for c in self.columns_names]
+        if lower_case_columns:
+            if is_details:
+                filtered_columns = [CLUSTER_ID_COL, TIME_ID_COL] + [
+                    c for c in df.columns.tolist() if any(regex in c.lower() for regex in lower_case_columns)
+                ]
+            elif self.mc_root == MCRoot.MC_ALL:
```

Comment:
I don't really understand why you separate MC_ALL from MC_IND here.
For me you should do the same thing for both (the MC_IND way) but I'm perhaps missing something.

Reply:
The first if is not for `MC_IND`

If you mean `regex` vs exact `match`, I thought that `regex` makes much sense for the `MC_ALL`

Reply:
What I meant was that I think you can remove the `elif self.mc_root == MCRoot.MC_ALL: ` check but I'm perhaps not understanding something

Reply:
I don't understand you either, I told that I prefer keeping the regex search for `MC_ALL`.
If you remove the `elif` you get an `exact match` for the `MC_ALL`.

Reply:
Okay. But then why do you want an exact match for MC_IND ?

Reply:
Okay, I'll see with you on Monday for the thing you've noticed for MC_ALL it will be easier to understand.